website/src/pages/product/code-learning.tsx

```diff
@@ -0,0 +1,212 @@
+                                for both new hires and existing teammates.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-md-4 mb-4">
+                        <IconItem color="red" icon={FileSearchOutlineIcon} className="py-3">
+                            <p className="text-sans-serif">
+                                It’s challenging to easily explore and understand the content and structure of new and
+                                unfamiliar codebases.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-md-4 mb-4">
+                        <IconItem color="red" icon={HelpIcon} className="py-3">
+                            <p className="text-sans-serif">
+                                New engineering hires don’t know where to look for specific code in large and
+                                distributed libraries and repositories.
+                            </p>
+                        </IconItem>
+                    </div>
+                </div>
+                <div className="row justify-content-center py-4">
+                    <div className="col-lg-10 text-center">
+                        <a href="/blog/convoy-improved-their-developer-on-boarding-with-sourcegraph">
+                            <img src="/external-logos/convoy-logo.svg" />
+                        </a>
+                        <blockquote className="blockquote case-studies__quote case-studies__quote--in-content">
+                            <p>
+                                For our new developers, Sourcegraph has been invaluable to get to know the repository
+                                structure, to track down where code lives, and self-service during their investigations.
+                            </p>
+                            <footer className="blockquote-footer">Owen Kim, Senior Software Engineer, Convoy</footer>
+                        </blockquote>
+                    </div>
+                    <div className="col-lg-10 mt-2 py-4 text-center">
+                        <h5>
+                            Learn more about how{' '}
+                            <a href="/blog/convoy-improved-their-developer-on-boarding-with-sourcegraph">
+                                Convoy uses Sourcegraph to onboard new developers.
+                            </a>
+                        </h5>
+                    </div>
+                </div>
+            </ContentSection>
+            <ContentSection color="white" className="pb-2">
+                <h2 id="features" className="display-4 mb-5 text-center">
+                    Why learning code is faster with Sourcegraph
+                </h2>
+                <p className="text-center mb-5">
+                    Sourcegraph <a href="/universal-code-search">Universal Code Search</a> lets you explore across your
```

Comment:
This is a very long sentence. It also is from our POV. Can we shorten it and rephrase it to be more like "With Sourcegraph, you can..." instead of "Sourcegraph lets you"?